data/l10n/bzflag_sk.po

```diff
@@ -0,0 +1,3205 @@
+# Slovak translation of BZFlag
+
+msgid "xx"
+msgstr "Zaklad prekladu"
```

Comment:
Instead of:
```po
msgid "xx"
msgstr "Zaklad prekladu"
```

This could be:
```po
msgid "sk"
msgstr "Slovak"
```

That way, in the language selection menu, it will show "Slovak" instead of "sk".

Reply:
thank you, fixed